config/crd/v1/bases/velero.io_backuprepositories.yaml

```diff
@@ -104,6 +104,34 @@ spec:
                 - Ready
                 - NotReady
                 type: string
+              recentMaintenanceStatus:
+                description: RecentMaintenanceStatus is status of the recent repo
+                  maintenance.
+                items:
+                  properties:
+                    completeTimestamp:
```

Comment:
Seems the timestamp is also updated if the maintenance is failed.  This causes some inconsistency with the description in `lastMaintenanceTime`, maybe we should revise the description in `lastMaintenanceTime` to stress it's time for last successful run or completed.

Reply:
Yes, we have done that in the comment of `lastMaintenanceTime` in the CRD. See this comment https://github.com/vmware-tanzu/velero/pull/8532#discussion_r1891344834.

Reply:
Changed the comment in the CRD to "LastMaintenanceTime is the last time repo maintenance succeeded"